modules/account-lib/test/unit/mpc/dotTss.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @prettier
+ */
+import 'should';
+import { KeyPair, TransferBuilder } from '../../../src/coin/dot';
+
+import Eddsa from '../../../src/mpc/tss';
+import { buildTestConfig } from '../coin/dot/transactionBuilder/base';
+import utils from '../../../src/coin/dot/utils';
+
+describe('TSS EDDSA key generation and signing', () => {
+  it('should produce valid signature', async function () {
+    const config = buildTestConfig();
+    const material = utils.getMaterial(config);
+
+    const MPC = await Eddsa();
+    const A = MPC.keyShare(1, 2, 3);
+    const B = MPC.keyShare(2, 2, 3);
+    const C = MPC.keyShare(3, 2, 3);
+
+    const A_combine = MPC.keyCombine(A.uShare, [B.yShares[1], C.yShares[1]]);
+    const B_combine = MPC.keyCombine(B.uShare, [A.yShares[2], C.yShares[2]]);
+    // const C_combine = MPC.keyCombine(C.uShare, [A.yShares[3], B.yShares[3]]);
+
+    const dotKeyPair = new KeyPair({ pub: A_combine.pShare.y });
+
+    const builder = new TransferBuilder(config).material(material);
+    builder
+      .amount('90034235235322')
+      .to({ address: '5Ffp1wJCPu4hzVDTo7XaMLqZSvSadyUQmxWPDw74CBjECSoq' })
+      .sender({ address: dotKeyPair.getAddress() })
+      .validity({ firstValid: 3933, maxDuration: 64 })
+      .referenceBlock('0x149799bc9602cb5cf201f3425fb8d253b2d4e61fc119dcab3249f307f594754d')
+      .sequenceId({ name: 'Nonce', keyword: 'nonce', value: 200 })
+      .fee({ amount: 0, type: 'tip' });
+    const tx = await builder.build();
+
+    // tx.toBroadcastFormat() generates a signing payload to be signed
+    // construct.signingPayload(this._dotTransaction, {
+    //   registry: this._registry,
+    // });
+
+    // create a buffer out of the txHex
+    const message_buffer = Buffer.from(tx.toBroadcastFormat(), 'utf-8');
```

Comment:
```suggestion
    const message_buffer = Buffer.from(tx.toBroadcastFormat(), 'hex');
```
from https://github.com/paritytech/txwrapper-core/blob/main/packages/txwrapper-core/src/core/construct/createSigningPayload.ts#L63 looks like they returned a hex